proxy/src/main/java/com/wavefront/agent/preprocessor/AgentPreprocessorConfiguration.java

```diff
@@ -80,7 +80,7 @@ public void loadFromStream(InputStream stream) {
             PreprocessorRuleMetrics ruleMetrics = new PreprocessorRuleMetrics(
                 Metrics.newCounter(new TaggedMetricName("preprocessor." + ruleName, "count", "port", strPort)),
                 Metrics.newCounter(new TaggedMetricName("preprocessor." + ruleName, "cpu_nanos", "port", strPort)),
-                Metrics.newCounter(new TaggedMetricName("preprocessor." + ruleName, "checked.count", "port", strPort)));
+                Metrics.newCounter(new TaggedMetricName("preprocessor." + ruleName, "checked-count", "port", strPort)));
```

Comment:
This change is needed to avoid confusion when users query `~proxy.preprocessor.*.count`